tracker-edtracker/ftnoir_tracker_edtracker.cpp

```diff
@@ -0,0 +1,71 @@
+#include "ftnoir_tracker_edtracker.h"
+#include "api/plugin-api.hpp"
+#include "compat/math.hpp"
+#include <cinttypes>
+#include <algorithm>
+#include <cmath>
+
+static const int max_value[] = {
+    100,
+    100,
+    100,
+    180,
+    90,
+    180
+};
+
+tracker_edtracker::tracker_edtracker() : pose { 0, 0, 0, 0, 0, 0 } {
+}
+
+tracker_edtracker::~tracker_edtracker() {
+    requestInterruption();
+    wait();
+    if (idevice > 0) close(idevice);
+}
+
+module_status tracker_edtracker::start_tracker(QFrame *) {
+    QString device = QString("/dev/input/js%1").arg(s.dev);
```

Comment:
Use `/dev/input/by-id/*`. The exact `js%d` value depends on the order in which the device was plugged in, rather than device name or serial number.